### ISSUE TITLE
Get-EntraGroupMember doesn't return service principals as members

### ISSUE THREAD

**Issue by alexandair** [microsoftgraph/entra-powershell#882]
As you can see in the following code snippets, AzureAD command returns the correct output while Entra command doesn't output service principal members:
```powershell
Get-AzureAdGroup -ObjectId 'b0c07cae-5f21-49db-b4a3-c42f53b98a8d' | 
Get-AzureAdGroupMember | 
Select-Object DisplayName,ObjectType

DisplayName        ObjectType      
-----------        ----------
Microsoft Azure    ServicePrincipal
Aleksandar Nikolic User
Alex Wilber        User
testgroup1         Group
Diegos App3        ServicePrincipal
```

```powershell
Get-EntraGroup -ObjectId 'b0c07cae-5f21-49db-b4a3-c42f53b98a8d' |
 Get-EntraGroupMember |
 Select-Object DisplayName,'@odata.type'

displayName        @odata.type
-----------        -----------
Aleksandar Nikolic #microsoft.graph.user
Alex Wilber        #microsoft.graph.user
testgroup1         #microsoft.graph.group
```

Problem lies in v1.0 version of Microsoft Graph API and `Get-MgGroupMember` used under the covers.
It's a known issue that's not fixed for months (years?).
Strangely, `Get-MgGroupMemberAsServicePrincipal`works as expected.

![image](https://github.com/microsoftgraph/entra-powershell/assets/3135708/9425a322-11c3-41a2-bfef-bf5ce38f5005)

**Comment by snehalkotwal** [comment]
@alexandair Thanks for raising the issue, We are looking into it.

**Comment by alexandair** [comment]
@SteveMutungi254 @KenitoInc 

I see this issue is closed, but it's still not fixed in 0.12.0.
